### DISCUSSION TITLE
粒子效果除了导出bmp还有其他方式吗？

### DISCUSSION THREAD

**Discussion by zhengcling** [Tencent/libpag#2199] [Q&A] [answered]
粒子效果除了导出bmp还有其他方式吗？

当前导出bmp在部分Chrome浏览器中偶现播放到一半，卡住问题；以及已知的微信浏览器自动播放问题

有没有其他方式可以规避？比如说先导出图片，然后转成pag等等？

**Comment by kevingpqi123** [comment, marked as answer]
BMP 预合成支持导出成两个格式：视频序列和位图序列，导出成位图序列看下，但此时文件会偏大，性能也会受影响
